examples/ui.apps/src/content/jcr_root/apps/core-components-examples/clientlibs/clientlib-site/styles/components/demo.less

```diff
@@ -46,6 +46,7 @@
 
 .cmp-examples-demo__component--widthFull {
     width: 100%;
+    max-width: 100%;
```

Comment:
Isn't redundant to have `width: 100%` and `max-width: 100%` on the same element?